src/modules/organisations/organisations.service.ts

```diff
@@ -71,4 +71,8 @@ export class OrganisationsService {
       throw new InternalServerErrorException(`An internal server error occurred: ${error.message}`);
     }
   }
+
+  async findById(id: string): Promise<Organisation> {
```

Comment:
Put a try/catch around your services or controllers where you are making database calls